algrtm/src/components/PianoRollDescription.js

```diff
@@ -0,0 +1,33 @@
+import Image from "next/image";
+import React from "react";
+
+const PianoRollDescription = () => {
+  return (
+    <section className="flex flex-col items-center justify-center w-full px-[12.625rem] mt-[3.94rem]">
+      <div className="flex gap-[4.5rem] items-center justify-center">
+        <div className="flex flex-col gap-4 max-w-[24.03rem] w-full">
+          <h1 className="h1 text-neutralLight-neutral10 leading-[6.02rem]">
+            PianoRoll
+          </h1>
+          <div>
+            <p className="h6 text-neutralLight-neutral20 leading-[1.98rem]">
+              At its surface, PianoRoll serves as a platform for pianists to
+              track and share their progress with friends and the broader
+              community. Yet, it’s so much more than that.
+            </p>
```

Comment:
The phrase `platform for pianists to track and share their progress` should have `font-weight: 600`

![image](https://github.com/Nospoko/algrtm/assets/26410548/52dd5a51-9cf9-436f-9890-8bb3a9025550)